docs/README.md

```diff
@@ -0,0 +1,41 @@
+# Brave docs
+
+This directory contains documentation for the Brave Browser.
+
+**If you add new documents, please also add a link to them in the Document Index
+below.**
```

Comment:
yeah right 😄. Everyone will forget this.

The only way I see this can work is to add a presubmit to enforce `docs/README.md` modification if a file was added into `docs/`.